apps/OpenSign/public/locales/fr/translation.json

```diff
@@ -0,0 +1,591 @@
+    "tagline-help":"Ce slogan apparaîtra sur votre profil public OpenSign par exemple. https://opensign.me/alex",
+    "welcome":"Content de te revoir!",
+    "Login-to-your-account" :"Se connecter à son compte",
+    "password" :"Mot de passe",
+    "forgot-password":"Mot de passe oublié?",
+    "loading":"Chargement...",
+    "of":"de",
+    "sign-SSO":"Connectez-vous avec SSO",
+    "login-page":"Page de connexion",
+    "or":"OU",
+    "signup-page":"Page d'inscription",
+    "password-length":"Le mot de passe doit être 8 Longs caractères",
+    "password-case":"Le mot de passe doit contenir lettre majuscule, lettre minuscule, chiffre",
+    "password-special-char":"Le mot de passe doit contenir un caractère spécial",
+    "agreee":"je suis d'accord avec le",
+    "term":"Conditions d'utilisation",
+    "subscribe-to-opensign":"Abonnez-vous à la newsletter OpenSign",
+    "register" :"Registre",
+    "sidebar": {
+    "Dashboard": "Tableau de bord",
+    "Sign yourself": "signez vous-même",
+    "Request signatures":"Demander des signatures",
+    "Templates":"Modèles",
+    "Templates-Children":{
+    "Create template":"Créer un modèle",
+       "Manage templates": "Gérer les modèles"
+    },
+    "OpenSign™ Drive":"Lecteur OpenSign™",
+    "Reports":"Rapports",
+    "Reports-Children" :{
+      "Need your sign": "Besoin de signer",
+      "In Progress": "En cours",
+      "Completed":"Complété",
+      "Drafts": "Brouillons",
+      "Declined": "a refusé",
+      "Expired":"Expiré",
+      "Contactbook":"Carnet de contacts"
+    },
+    "Settings": "Paramètres",
+    "Settings-Children":{
+    "My Signature": "Ma signature",
+    "API Token":"Jeton API",
+    "Webhook":"Webhook",
+    "Teams":"Équipes",
+    "Users":"Utilisateurs"
+      }
+    },
+    "dashboard-card":{
+    "Need your Signature":"Besoin de votre signature",
+    "Out for signatures":"En quête de signatures"
+      },
+   "report-name":{
+      "Draft Documents":"Projets de documents",
+      "Need your sign":"Besoin de votre signe",
+      "In-progress documents":"Documents en cours",
+      "Completed Documents":"Documents complétés",
+      "Declined Documents":"Documents refusés",
+      "Expired Documents":"Documents expirés",
+      "Recently sent for signatures":"Récemment envoyé pour signature",
+      "Recent signature requests":"Demandes de signature récentes",
+      "Drafts":"Brouillons",
+      "Contactbook":"Carnet de contacts",
+      "Templates":"Modèles",
+      "Teams":"Teams",
+      "Users":"Utilisateurs"
+    },
+    "report-heading":{
+      "Sr.No":"Sr. Non",
+      "Title":"Titre",
+      "Email":"E-mail",
+      "Phone":"Téléphone",
+      "File":"Fichier",
+      "Owner":"Propriétaire",
+      "Signers":"Signataires",
+      "Note": "Note", 
+      "Folder":"Dossier",
+      "Reason": "Raison",
+      "Parent Team":"Équipe de parents",
+      "Active":"Actif",
+      "Role":"Rôle",
+      "Team":"Équipe",
+      "Name":"Nom",
```

Comment:
"name":"Nom et Prénom",

User can missunderstand and only write his Name and not firstname